planning/mission_planning/mission_planner/package.xml

```diff
@@ -17,6 +17,9 @@
   <depend>tf2_ros</depend>
   <depend>tf2_geometry_msgs</depend>
 
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
```

Comment:
I think we can change this to `ament_cmake_cppcheck` in case the linter checks fail due to anything other than cppcheck

Reply:
@jilaada I've removed `ament_lint_common` and added `ament_cmake_cppcheck` in c101295b. I fixed the linting errors when the rest of the linters were enabled, should I revert those changes and only fix those that `cppcheck` detects?

Reply:
If you've already made those fixes it's a bonus so its better to just leave them. Thanks for changing it to `ament_cmake_cppcheck` though!